advocacy_docs/edb-postgres-ai/migration-etl/index.mdx

```diff
@@ -15,15 +15,27 @@ The available migration tools are suitable for different migration scenarios. Th
 
 ## EDB Data Migration Service
 
-[EDB Data Migration Service](data-migration-service) is a cloud-based service that helps you migrate your data to an [EDB Hosted Cloud Service](/edb-postgres-ai/cloud-service/getting_started/planning/choosing_your_deployment/edb_hosted_cloud_service/) database.
+[EDB Data Migration Service](data-migration-service) is a cloud-based, Hybrid Control Plane service that helps you migrate your data, either with **Snapshot** or **Snapshot + Streaming** capabilities.
 
-EDB DMS offers a secure and fault-tolerant way to migrate database data to the EDB Postgres AI platform using change data capture and event streaming technologies. This allows the migration to stream any changes in the source database to the EDB Cloud Service database until you decide to cut over.
+EDB DMS offers a secure and fault-tolerant way to migrate database data using change data capture and event streaming technologies. This allows the migration to stream any changes in the source database to the destination database until you decide to cut over.
 
-At the end of a data migration with EDB DMS, the data in your source database will have been copied to a Postgres database managed with EDB Postgres AI Cloud Service.
+At the end of a data migration with EDB DMS, the data in your source database will have been copied to a Postgres database.
 
-- **Purpose:** Migration of database data
+### Migration scenarios
 
-- **Applicable migration scenarios:** From Oracle, PostgreSQL or EDB Postgres Advanced Server to [EDB Hosted Cloud Service](/edb-postgres-ai/cloud-service/getting_started/planning/choosing_your_deployment/edb_hosted_cloud_service/) databases with PostgreSQL, EDB Postgres Advanced Server or EDB Postgres Extended Server
+**To HCP-managed databases**
+
+- **Purpose:** Migration of database data to an HCP-managed Postgres database
+
+- **Applicable migration scenarios:** From Oracle, PostgreSQL or EDB Postgres Advanced Server to EDB Postgres AI Hybrid Control Plane databases with PostgreSQL, EDB Postgres Advanced Server or EDB Postgres Extended Server
+
+** To self-managed Postgres databases**
+
+Alternatively, you can use the EDB Data Migration Service to migrate either HCP-managed databases or other external, self-managed databases to a different Postgres destination database outside of the Hybrid Control Plane.
+
+- **Purpose:** Migration of database data to a self-managed Postgres database
+
+- **Applicable migration scenarios:** From HCP-managed or self-managed databases running on Oracle, PostgreSQL or EDB Postgres Advanced Server to self-managed instances of PostgreSQL, EDB Postgres Advanced Server or EDB Postgres Extended Server
```

Comment:
Saying "to self-managed instances of ..." may not be indicative of the full capability.  Technically speaking, I think it should be possible to target other CSPs' fully managed Postgres clusters.  That said, it's probably ok to not explicitly point that out as a capability here in the  documentation since that would be supporting migrations to competitors' databases. If that situation gets asked about in a support or sales situation, we can address it then.